### PR TITLE
Task/805 Explain header styling in CSS section of Style Guide

### PR DESCRIPTION
**Pull request by pieer**

### CODE REVIEW

**Comment by sandrina-p** [comment]
Oh, I addressed this directly on #563 Contributions Widget PR (as I said on the issue thread), but I like this too! Contributions PR will be merged today, this could be merged after.

File: docs/Style-Guide.md [hunk 1]
```diff
@@ -44,6 +44,22 @@ We strive to keep semantics (HTML) decoupled from styling (CSS). For example, wh
 </style>
 ```
 
+### Titles
+
+There should not be heading gap in the page.
+So for example, h3 can be present only if there is an h2 tag.
+But sometime a title look better in the page if h2 is the size of h3. In that case we will use
```
Comment (sandrina-p) [inline docs/Style-Guide.md]: heading gap -> headings gap
but sometime -> but sometimes

**Review by sandrina-p — APPROVED** [review]
Good improvement!

**Review by taoeffect — APPROVED** [review]
Great job @pieer!

**Pull request merged**